.CI/build_all

```diff
@@ -25,4 +27,4 @@ git ls-tree --name-only master -- . | xargs -I {} sh -c "rm -rf {} && echo Remov
 popd > /dev/null
 
 # We just want to build all of the recipes.
-conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5"
+conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
```

Comment:
The PR now also does not contain the workaround anymore, because I think the conda-build-all changes have been merged and released, right?